nilearn/plotting/surf_plotting.py

```diff
@@ -369,30 +381,36 @@ def _get_cmap_matplotlib(cmap, vmin, vmax, threshold=None):
 
 
 def _compute_facecolors_matplotlib(bg_map, faces, n_vertices,
-                                   darkness, alpha):
+                                   darkness, alpha, scale_bg_map):
     """Helper function for plot_surf with matplotlib engine.
 
     This function computes the facecolors.
     """
     # set alpha if in auto mode
     if alpha == 'auto':
         alpha = .5 if bg_map is None else 1
+
     if bg_map is None:
         bg_data = np.ones(n_vertices) * 0.5
     else:
         bg_data = load_surf_data(bg_map)
         if bg_data.shape[0] != n_vertices:
             raise ValueError('The bg_map does not have the same number '
                              'of vertices as the mesh.')
+
     bg_faces = np.mean(bg_data[faces], axis=1)
-    if bg_faces.min() != bg_faces.max():
-        bg_faces = bg_faces - bg_faces.min()
-        bg_faces = bg_faces / bg_faces.max()
+    if scale_bg_map:
+        if bg_faces.min() != bg_faces.max():
```

Comment:
maybe `if scale_bg_map and bg_faces.min() != bg_faces.max()`?

Reply:
:+1: done